edx_exams/apps/core/signals/handlers.py

```diff
@@ -65,3 +66,17 @@ def listen_for_exam_attempt_errored(sender, signal, **kwargs):  # pylint: disabl
         event_data={'exam_attempt': kwargs['exam_attempt']},
         event_metadata=kwargs['metadata'],
     )
+
+
+@receiver(EXAM_ATTEMPT_RESET)
+def listen_for_exam_attempt_reset(sender, signal, **kwargs):  # pylint: disable=unused-argument
+    """
+    Publish EXAM_ATTEMPT_RESET signal onto the event bus
+    """
+    get_producer().send(
+        signal=EXAM_ATTEMPT_RESET,
+        topic='exam-attempt-reset',
```

Comment:
I think we want to use the `learning-exam-attempt-lifecycle` topic that Alie created in https://github.com/edx/terraform/pull/6144. Maybe we can merge in https://github.com/edx/edx-exams/pull/193 first and you can rebase?

Reply:
yeah I can wait on this and grab that value after it's merged

Reply:
Sounds good. Alie's pull request was merged, so I can re-review once you've rebased.